lib/internal/errors.js

```diff
@@ -134,11 +134,14 @@ E('ERR_INVALID_TUPLE', '%s must be an iterable %s tuple');
 E('ERR_INVALID_URL', 'Invalid URL: %s');
 E('ERR_INVALID_URL_SCHEME',
   (expected) => `The URL must be ${oneOf(expected, 'scheme')}`);
+E('ERR_INVALID_REPL_HISTORY', 'Expected array, got %s');
 E('ERR_IPC_CHANNEL_CLOSED', 'channel closed');
 E('ERR_IPC_DISCONNECTED', 'IPC channel is already disconnected');
 E('ERR_IPC_ONE_PIPE', 'Child process can have only one IPC pipe');
 E('ERR_IPC_SYNC_FORK', 'IPC cannot be used with synchronous forks');
 E('ERR_MISSING_ARGS', missingArgs);
+E('ERR_PARSE_HISTORY_DATA',
+  (oldHistoryPath) => `Could not parse history data in ${oldHistoryPath}`);
```

Comment:
Just a nit, but it's a bit odd to have one of the new codes use `%s` while the other uses a template string.